<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add House</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Postal Code:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>House Size (mÂ²):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Register:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_lineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="country_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="size_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="postalcode_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="state_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="city_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="address2_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="address1_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imageLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>250</width>
     <height>230</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>230</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chooseButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>load_image()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>add_house_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>load_image()</slot>
  <slot>add_house_clicked()</slot>
 </slots>
</ui>
